tgg/seqr_scripts/pull_seqr_pedigrees.py

```diff
@@ -4,12 +4,16 @@
 import requests
 
 logging.basicConfig(
-    format="%(asctime)s): %(message)s", datefmt="%m/%d/%Y %I:%M:%S %p",
+    format="%(asctime)s): %(message)s",
+    datefmt="%m/%d/%Y %I:%M:%S %p",
 )
 logger = logging.getLogger()
 logger.setLevel(logging.INFO)
 
 SEQR_URL = "https://seqr.broadinstitute.org"
+GET_FAMILIES_URL = "get_families"
+GET_INDIVIDUALS_URL = "get_individuals"
+
 
 def pull_project_peds(session_id: str, projects: set):
```

Comment:
also not in the scope of this PR, but this should likely be
```
def pull_project_peds(session_id: str, projects: Set[str]):
```
(with `from typing import Set` at the top)